### ISSUE TITLE
Catch URL formatting issues in DESCRIPTION?

### ISSUE THREAD

**Issue by cboettig** [r-lib/roxygen2#1420]
Some packages have historically used annotations on the URL field, such as `curl`:

```
URL: https://github.com/jeroen/curl (devel) 
    https://curl.se/libcurl/ (upstream)
BugReports: https://github.com/jeroen/curl/issues
```

(https://github.com/jeroen/curl/blob/e7502d27c7cff17baa0930c3327f84ae4918aa24/DESCRIPTION#L22-L23)

This syntax is no longer acceptable on CRAN, resulting in warnings such as:

```
   Found the following (possibly) invalid URLs:
     URL: https://docs.ropensci.org/gbifdb/ (website)
https://github.com/ropensci/gbifdb
       From: man/gbifdb-package.Rd
       Status: Error
       Message: URL using bad/illegal format or missing URL
       URL contains spaces
     Spaces in an http[s] URL should probably be replaced by %20
```

but will not be caught by a standard R CMD check (on R <= 4.2.0) or by `urlchecker`.  Would it be reasonable to add such a check to `urlchecker`?  (The correct syntax apparently cannot have non-URLs in the comma-separated list I think).

**Comment by cboettig** [comment]
Thanks @gaborcsardi & @jennybc !  Does this need to be reported separately for RStudio?  It is still creating broken links in the package window there:

![image](https://user-images.githubusercontent.com/222586/185497514-5bb797fc-c6a1-41d6-b9cd-cf4688615ddc.png)

**Comment by gaborcsardi** [comment]
@cboettig Yes, please report it at rstudio/rstudio. Thanks!

**Comment by hadley** [comment]
Sounds like this was resolved via a fix to desc?

**Comment by jennybc** [comment]
No, I suspect this still needs a fix in roxygen2, but the smarts already in desc will make it less painful to fix.
